crates/esthri-cli/src/http_server.rs

```diff
@@ -114,12 +113,8 @@ impl ErrorTracker {
         }
     }
     fn record_error(error_tracker: ErrorTrackerArc, err: anyhow::Error) {
```

Comment:
While touching this function, please add a comment describing the role of the function and the meaning of the parameters.

I have yet to find a standard for Rust documentation. If you are aware of one, or one that we as a team use, would you please link it (here or slack)

Reply:
This is the standard for Rust documentation: https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html

Reply:
Updated